modules/installing-rhv-requirements.adoc

```diff
@@ -30,6 +30,23 @@ If you increase the number of virtual machines in the {rh-virtualization} enviro
 *** 16 GiB or more for each of the three control plane machines which provide the control plane.
 *** 16 GiB or more for each of the three compute machines, which run the application workloads.
 * The {rh-virtualization} storage domain must meet link:https://access.redhat.com/solutions/4770281[these etcd backend performance requirements].
+ifeval::["{context}" == "installing-rhv-default"]
+* For affinity group support:
+Three or more hosts in the {rh-virtualization} cluster. If necessary, you can disable affinity groups.
+See xref:../../installing/installing_rhv/installing-rhv-customizations.adoc#installing-rhv-example-install-config-yaml_installing-rhv-customizations[Example: Removing all affinity groups for a non-production lab setup].
```

Comment:
We still can't use xrefs for now in a module, so this link should be moved to an `.Additional resources` list directly following the module at the assembly level (see last sentence in this [NOTE](https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#writing-assemblies)). Alternatively, can remove the sentence or just make the section/example title reference be in double-quotations (per IBM SG) with no `xref`.

Reply:
I put the xref in .Additional resources in the assembly.